cli/things.go

```diff
@@ -152,7 +152,7 @@ var cmdThings = []cobra.Command{
 	cobra.Command{
 		Use:   "connections",
 		Short: "connections <thing_id> <user_auth_token>",
-		Long:  `List of Channels connected to Thing`,
+		Long:  `List of Channels connected to Things`,
```

Comment:
Same here, list of channels connected to one thing. So it should be `Thing`.

Reply:
right